test/sampler.jl

```diff
@@ -59,6 +60,12 @@ end
                     Random.seed!(1)
                     samples, stats = sample(h, τ, θ_init, n_samples; verbose=false, progress=PROGRESS)
                     @test mean(samples[n_adapts+1:end]) ≈ zeros(D) atol=RNDATOL
+                    if "GEWEKE_TEST" in keys(ENV) && ENV["GEWEKE_TEST"] == "1"
```

Comment:
@yebai This is the main change. When the test is run with `GEWEKE_TEST=1` (on GitHub Actions), Geweke tests will be run and the Q-Q plot will be displayed (using UnicodePlots.jl).